main/FatesHistoryInterfaceMod.F90

```diff
@@ -2386,6 +2391,11 @@ subroutine update_history_dyn(this,nc,nsites,sites,bc_in)
                hio_seedling_pool_si    => this%hvars(ih_seedling_pool_si)%r81d, &
                hio_seeds_in_si         => this%hvars(ih_seeds_in_si)%r81d, &
                hio_seeds_in_local_si   => this%hvars(ih_seeds_in_local_si)%r81d, &
+               hio_seed_bank_si_pft    => this%hvars(ih_seed_bank_si_pft)%r82d, &
```

Comment:
These might need to be moved below line 2611 because of line continuation issues with the nag compiler. Not totally sure.

Reply:
Good catch @JessicaNeedham.  I've made the update.